tasks/eslint.js

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.config('eslint', {
+    options: {
+      eslintrc: '.eslintrc'
+    },
+    app: [
+      'bin/*.js', 'tasks/*.js', 'templates/*.js', 'test/*.js'
```

Comment:
Oh, also probably need to do a glob of `'test/**/*.js'` here (instead of just `'test/*.js'`), so we get the nested test/local/*.js files.